src/Statistics/BinomialConfidenceInterval.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Statistics/BinomialConfidenceInterval.h"
+
+#include <cmath>
+#include <cstdint>
+
+namespace orbit_statistics {
+
+double sqr(double x) { return x * x; }
+
+[[nodiscard]] BinomialConfidenceInterval WilsonBinomialConfidenceIntervalEstimator::Estimate(
+    float ratio, uint32_t trials) const {
+  if (trials == 0) {
+    return {0.0f, 1.0f};
+  }
```

Comment:
Since you're already validating preconditions here, what about ratio?

Reply:
> e. g. throw

If yes, then we use `ORBIT_CHECK` which aborts (we don't use exceptions).